drivers/hud_twenty_twenty_to_twenty_twenty_two/app/models/hud_twenty_twenty_to_twenty_twenty_two/disability/csv.rb

```diff
@@ -0,0 +1,30 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+require 'kiba-common/sources/csv'
+require 'kiba-common/destinations/csv'
+
+module HudTwentyTwentyToTwentyTwentyTwo::Disability::Csv
+  module_function
+
+  def up(source_name, destination_name)
+    HudTwentyTwentyToTwentyTwentyTwo::Disability::Transform.up(
+      Kiba::Common::Sources::CSV,
+      {
+        filename: source_name,
+        csv_options: {
+          headers: :first_row,
+          skip_blanks: true,
+        },
+      },
+      Kiba::Common::Destinations::CSV,
+      {
+        filename: destination_name,
+        headers: GrdaWarehouse::Hud::Disability.hmis_configuration(version: '2022').keys.map(&:to_s),
+      },
+    )
+  end
```

Comment:
Is it worth making a base class for these so each of them only has a method to return the relevant class name?  like:
```
private def hmis_class_name
  'Disability'
end
````
Unless I'm missing something all of these CSV files are identical otherwise.
Alternatively you could provide the full warehouse class name and full csv class name so there wasn't any crazy interpolation.

Reply:
or maybe it's `def headers` and `def transformer_class`